src/APIFunctions/User.js

```diff
@@ -11,11 +11,11 @@ let GENERAL_API_URL = process.env.REACT_APP_GENERAL_API_URL
  * @returns {UserApiResponse} Containing any error information or the array of
  * users.
  */
-export async function getAllUsers(token, query = null, page = null) {
+export async function getAllUsers(token, query = null, page = null, sortColumn = null, sortOrder = null) {
```

Comment:
lets turn this function into accepting an object, that way the order of the params dont matter
```suggestion
export async function getAllUsers({
    token,
    query = null,
    page = null,
    sortColumn = null,
    sortOrder = null,
}) {
```

Reply:
resolved with @syng828
[changed getAllUsers function into accepting object + use url class to…](https://github.com/SCE-Development/Clark/pull/1402/commits/2761592fda73f52b539f4f3796b2df6acac05fe6)